### PR TITLE
Form

### PR DESCRIPTION
**Pull request by Mehria-Rahmani**
-create a contact form for my project.

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Mehria-Rahmani , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a contribution from one person yet is a group project, kindly make sure everybody contributes to the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by jaflih — COMMENTED** [review]
Hi @Mehria-Rahmani , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that is pair programming, so both students should contribute.

Kindly make sure everybody in your team contributes to the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -226,5 +226,46 @@ <h2>About Me</h2>
             </div>
         </section>
 
+        <section class="contact">
+            <div class="contact-container">
+            <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            <form>
```
Comment (elyor-doniyorov) [inline index.html]: - [ ] Kindly use in your form tag the `action `and `method `attribute to integrate with [formspree service](https://formspree.io/html) 👇 
* `action` value should be similar to `https://formspree.io/f/{form_id}`
* `method` value equals `POST`;

File: index.html [hunk 2]
```diff
@@ -226,5 +226,46 @@ <h2>About Me</h2>
...
+               <ul>
+            <li><input name="name"  id="name" required type="text" placeholder="Full name"><hr></li>
+            <li><input name="email-address"  id="email" required type="email" placeholder="Email address"><hr></li>
+            <li><textarea name="message" id="message" placeholder="Enter text here"></textarea></li>
```
Comment (elyor-doniyorov) [inline index.html]: - [ ] Also please make sure that `name `input has `maxlength="30"` attribute and `message `input has `maxlength="500"` attribute.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @Mehria-Rahmani, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔️ There are no linter errors
✔️ All input field have required attribute
✔️ etc.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- to add a good `description`, a `screenshot`, `how to setup` section, `issue page link` in your Readme file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -226,5 +225,46 @@ <h2>About Me</h2>
...
         </section>
 
+        <section class="contact">
+            <div class="contact-container">
+            <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            <form action="https://formspree.io/f/{form_id}" method="post">
+               <ul>
+            <li><input name="name"  id="name" required maxlength="30" type="text" placeholder="Full name"><hr></li>
+            <li><input name="email-address"  id="email" required type="email" placeholder="Email address"><hr></li>
+            <li><textarea name="message" id="message" minlength="500" placeholder="Enter text here"></textarea></li>
```
Comment (danmainah) [inline index.html]: - [ ] @Team good job so far, kindly check your attribute `minlength` , it supposes to be `maxlength`.Also, make sure your form submits data to your `Formspree` account.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi  @Team, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Team


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mehria-Rahmani** [comment]
Thank you so much

**Pull request merged**